jira.go

```diff
@@ -98,44 +104,70 @@ func openJiraTicket(endpointAPI string, orgID string, token string, jiraProjectI
 
 	ticket, err := json.Marshal(jiraTicket)
 	if err != nil {
-		log.Fatalln(err)
+		fmt.Println("Error while creating the ticket")
+		return emptyByteVar, errors.New("Failure, Failure to create ticket(s)")
 	}
 
-	responseData := makeSnykAPIRequest("POST", endpointAPI+"/v1/org/"+orgID+"/project/"+projectInfo.K("id").String().Value+"/issue/"+vulnID+"/jira-issue", token, ticket)
+	responseData, er := makeSnykAPIRequest("POST", endpointAPI+"/v1/org/"+orgID+"/project/"+projectInfo.K("id").String().Value+"/issue/"+vulnID+"/jira-issue", token, ticket)
 
-	emptyByteVar := make([]byte, 128)
+	if er != nil {
+		fmt.Println("Request failed")
+		return emptyByteVar, errors.New("Failure, Failure to create ticket(s)")
+	}
 
 	if bytes.Equal(responseData, emptyByteVar) {
-		return emptyByteVar, errors.New("Failure, snyk API request responseData is empty")
+		fmt.Printf("Request response from %s is empty\n", endpointAPI)
+		return emptyByteVar, errors.New("Failure, Failure to create ticket(s)")
 	}
 
 	return responseData, nil
 }
 
-func openJiraTickets(endpointAPI string, orgID string, token string, jiraProjectID string, jiraTicketType string, assigneeID string, labels string, projectInfo jsn.Json, vulnsForJira map[string]interface{}, priorityIsSeverity bool) (string, error) {
+func openJiraTickets(endpointAPI string, orgID string, token string, jiraProjectID string, jiraTicketType string, assigneeID string, labels string, projectInfo jsn.Json, vulnsForJira map[string]interface{}, priorityIsSeverity bool) (string, string, error) {
 	fullResponseDataAggregated := ""
+	fullListNotCreatedIssue := ""
+	alreadyTried := false
+	RequestFailed := false
 
 	for _, vulnForJira := range vulnsForJira {
+		RequestFailed = false
```

Comment:
I would probably use a counter instead of 2 booleans, this way I can count retries and try numerous times, with the actual number defined in a variable that can be overriden by ENV VAR.